docs/developer/index.md

```diff
@@ -32,3 +32,4 @@ respectively) are all that is needed.
 * [Code Structure and Organization](organization.md)
 * [Developer Tools](tools.md)
 * [Integrating RDycore with E3SM](e3sm.md)
+* [Description About Mesh](mesh.md)
```

Comment:
Mesh Description? Description of Mesh (or perhaps Description of Unstructured Grid)?

Reply:
Updated as suggested.